packages/ui/src/IconButton.tsx

```diff
@@ -117,7 +119,13 @@ export const IconButton = forwardRef(
             }}
           >
             <Icon color={iconColor} name={icon} prefix={prefix || "fas"} size={size} />
+            {indicator && (
+              <View style={{position: "absolute", bottom: "20%", right: "20%"}}>
+                <Icon color="primary" name="circle" prefix={prefix || "fas"} size="sm" />
```

Comment:
Do we want an indicator color too? Happy to stick with as-is and add that later though.

Reply:
Good call. I'm adding position in there while I'm at it